client/components/VerticalBar/VerticalBarActions.tsx

```diff
@@ -0,0 +1,8 @@
+import { ButtonGroup } from '@rocket.chat/fuselage';
+import React, { FC, memo } from 'react';
+
+const VerticalBarActions: FC<{ children: React.ReactNode }> = (props) => (
```

Comment:
Do not use "React.*", always import what you're referencing. React is to drop support to this pattern.

Reply:
Great observation! Was a lack of attention here!